### ISSUE TITLE
Can't start dev server on Windows

### ISSUE THREAD

**Issue by dv297** [cerner/terra-core#1509]
# Issue Description
After cloning the project, running `npm run clean:install` and `npm run build`, I'm unable to start the dev server using `npm start`

```
λ npm start

> terra-core@0.1.0 start C:\workspace\dv297_personal_repo\terra-core
> tt:serve --config webpack.config

The filename, directory name, or volume label syntax is incorrect.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! terra-core@0.1.0 start: `tt:serve --config webpack.config`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the terra-core@0.1.0 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```

## Issue Type

- [ ] New Feature
- [ ] Enhancement
- [X] Bug
- [ ] Other

## Expected Behavior
Should start dev server on Windows

## Current Behavior
Doesn't start dev server, shows above message. 

## Steps to Reproduce

1. Clone project on Windows machine
2. Run `npm run clean:install`
3. Run `npm start`

## Environment

* Component Version:
* Browser Name and Version: Google Chrome Version 66.0.3359.139 (Official Build) (64-bit)
* Operating System and version (desktop or mobile): Windows 10 Enterprise - Desktop

**Comment by emilyrohrbough** [comment]
@BenBoersma This is likely the issue you are working through with webpack.

**Comment by bjankord** [comment]
Need to follow up with @BenBoersma  on this

**Comment by BenBoersma** [comment]
Windows doesn't allow certain characters in the naming of files (in this case ':') and NPM was trying to create a file that linked to Terra-Toolkit's bin file "tt:serve". This fails silently during 'npm install'. 

This error should be fixed with Terra-Toolkit version 4.0.0 with the [renaming](https://github.com/cerner/terra-toolkit/pull/113/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R28) of "tt:serve" to "tt-serve" since that does not conflict Windows' naming conventions.

**Comment by dv297** [comment]
Looks like terra-toolkit was updated to 4.x.x, just checked out the project and was able to start the dev server, closing this issue, thanks!
